scripts/compaction.sh

```diff
@@ -0,0 +1,17 @@
+lineOfKeyspaces=`./service/bin/cqlsh -e "describe keyspaces; "`
+echo $lineOfKeyspaces
+
+for keyspace in ${lineOfKeyspaces[@]}
+do
+        echo "keyspace = '$keyspace'"
+        keyspaceWithoutQuotes="${keyspace//\"}"
+        echo "keyspaceWithoutQuotes = $keyspaceWithoutQuotes"
+        if [[ $keyspace == *"system"* || $keyspace == *"__simple_rf_test_keyspace__"* ]]; then
+                echo "Skipping $keyspace as this is a test or system keyspace!"
+                continue;
+        fi
+        txTable=" _transactions"
+        ./service/bin/cqlsh -e "use $keyspace; alter table \"_transactions\" with gc_grace_seconds = 0;"
+        ./service/bin/nodetool flush && ./service/bin/nodetool compact $keyspaceWithoutQuotes
```

Comment:
should `flush` also be passed the keyspace?

Reply:
No flush doesnt take a keyspace.

Reply:
http://docs.datastax.com/en/archived/cassandra/2.0/cassandra/tools/toolsFlush.html

Reply:
We can flush the entire memtable here, since this is run on all keyspaces. We will be a bit slower for the first keyspace but will save time on subsequent ones.

Reply:
we should only compact the `_transactions` table, not the whole keyspace. Doing the whole keyspace depending on the deployment and product can take over a whole day to run.

Reply:
I had transactions table in there but missed it when adding `kespaceWithoutQuotes`, fixing and testing again.